docs/release-notes/8.17.asciidoc

```diff
@@ -2,29 +2,54 @@
 == 8.17
 
 [discrete]
-[[release-notes-8.17.2]]
-=== 8.17.2
+[[release-notes-8.17.3]]
+=== 8.17.3
 
 [discrete]
-[[known-issue-8.17.2]]
+[[known-issue-8.17.3]]
 ==== Known issues
 
 // tag::known-issue[]
 [discrete]
-.{elastic-sec} crashes on {kib} instances with 1 GB of RAM on {ecloud} deployments
+.Duplicate alerts can be produced from manually running threshold rules 
 [%collapsible]
 ====
 *Details* +
-Whenever you open a page in {elastic-sec}, there's an attempt to install the {fleet} package with prebuilt rules. If the package hasn't been installed yet, {kib} starts downloading the latest version of it, then crashes with an `Out Of Memory` error. The process will then automatically restart and crash for the same reasons.
-
-This issue was discovered on February 6, 2025.
-
-*Workaround* +
-To resolve this issue, increase {kib}'s RAM to 2 GB.
+On November 12, 2024, it was discovered that manually running threshold rules could produce duplicate alerts if the date range was already covered by a scheduled rule execution.
+====
+// end::known-issue[]
 
+// tag::known-issue[]
+[discrete]
+.Manually running custom query rules with suppression could suppress more alerts than expected
+[%collapsible]
+====
+*Details* +
+On November 12, 2024, it was discovered that manually running a custom query rule with suppression could incorrectly inflate the number of suppressed alerts. 
 ====
 // end::known-issue[]
 
+[discrete]
+[[bug-fixes-8.17.3]]
+==== Bug fixes
+* Removes the option to sort IP ranges in the value list modal ({kibana-pull}210922[#210922]).
+* Fixes package name validation on the Automatic Import data stream page ({kibana-pull}210916[#210916], {kibana-pull}210770[#210770]).
+* Improves the way Automatic Import handles empty categorization results from LLMs ({kibana-pull}210420[#210420]).
+* Adds concurrency limits and request throttling to prebuilt rule routes ({kibana-pull}209551[#209551]).
+* Fixes a bug in {elastic-defend} for Linux where tty capture limit defaults were ignored.
+* Fixes a potential {elastic-defend} crash when generating multiple ransomware alerts on Windows. This issue was simultaneously mitigated by a cloud artifact update (manifest version 1.0.1381) on February 24, 2025. Internet-connected instances of {elastic-defend} will automatically receive this update -- no user intervention required. Air-gapped customers hosting their own artifacts should follow [these instructions](offline-endpoint). We would like to acknowledge [Todyl](https://www.todyl.com) for their assistance with this issue.
```

Comment:
The link on [these instructions] isn't working. The link on Todyl isn't either

Reply:
Fixed in [f6614ba](https://github.com/elastic/security-docs/pull/6550/commits/f6614bad97cf571d6680c477be4b6ab09b55d315)!